.github/workflows/org-inactive-user-management.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Delete Inactive Users in Github Organization'
+
+on:
+  schedule:
```

Comment:
The github action should get testable, if you enable trigger on PR: 
```
on:
  pull_request:
  schedule:
...
```

Later we could enable dry-run if the action was triggered by a PR.

Reply:
I would like to trigger this action manually like described [here](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) for testing. That is why i added the `workflow_dispatch` option. Is that also ok for you?